plugins/fluentd_telemetry_plugin/src/streamer.py

```diff
@@ -95,6 +96,18 @@ def ufm_telemetry_endpoints(self):
         xdr_ports_types = self.ufm_telemetry_xdr_ports_types.split(splitter)
         intervals = self.streaming_interval.split(splitter)
         msg_tags = self.fluentd_msg_tag.split(splitter)
+
+        error_message = ""
+        expected_amount = len(hosts)
+        for name, array in [("port", ports), ("url", urls), ("interval" ,intervals), ("message_tag_name", msg_tags),\
+                             ("xdr_mode", xdr_mode), ("xdr_ports_types", xdr_ports_types)]:
+            if len(array)!= expected_amount:
+                error_message += f"setting under ufm-telemetry-endpoint {name} for multi telemetry,"\
```

Comment:
instead of repeating the same message for all invalid fields, I suggest you to change it into one message that contains all the invalid fields after validating them
```suggestion
                setting under ufm-telemetry-endpoint for the following fields {invalid_fields} for multi telemetry,....etc
```

Reply:
agree, changed, but now I didnt write what they have, rather what the expected length they should be.